app/controllers/welcome_controller.rb

```diff
@@ -20,11 +20,7 @@ def search
     	@topics = Topic.search(@q, 10, is_fuzzy).to_a
       respond_to do |format|
         format.html {
-          if (@topics + @items).blank?
-            flash[:warning] = "#{@q}: No such topic found"
-            redirect_to new_topic_path(name: @q)
```

Comment:
redirecting to new_topic_path when user's intent is to search didn't make sense

Reply:
We need some way to add new topics though. Let's figure out a way to expose that somewhere.

Reply:
UX for adding a new topic needs to be better as well.